Homework2/TrainingManagementSystem/Training.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TrainingManagementSystem
+{
+    internal class Training
+    {
+        private string _descriptionText;
+        private Lecture _lecture;
+        private PracticalLesson _practicalLesson;
+        public Training(string description, Lecture lecture, PracticalLesson practicalLesson)
+        {
+            _descriptionText = description;
+            _lecture = lecture;
+            _practicalLesson = practicalLesson;
+        }
+        public string? DescriptionText { get; set; }
+
+        public Dictionary<Lecture, PracticalLesson> SetOfObjects { get; set; } = new Dictionary<Lecture, PracticalLesson>();
```

Comment:
Per requirement: The training should store a set of lectures and practice sessions in an **internal array**